Pod/Classes/TSSwiftMarkdownParser.swift

```diff
@@ -98,6 +100,19 @@ public class TSSwiftMarkdownParser: TSBaseParser {
         strongAndEmphasisAttributes = [NSFontAttributeName: strongAndEmphasisFont]
         
         if withDefaultParsing {
+            
+            addNumberedListParsingWithMaxLevel(0, leadFormattingBlock: { (attributedString, range, level) in
```

Comment:
I'm looking at where it's used and it seems like a sentinel to me: `let regexString = NSString(format: pattern, maxLevel > 0 ? "\(maxLevel)" : "") as String`. I think it should be optional instead so the intent is clear.